---
name: Diogo M. Camacho
position: "Senior Director"
aboutme: Computational Biology and Bioinformatics
address: "Rheos Medicines"
#phone: +1 617 432 7004
phone: +1 617 945 4383
email: "diogo.camacho.2008@gmail.com"
twitter: DiogoMCamacho
github: diogocamacho
linkedin: diogocamacho
headcolor: 414141
date: "`r format(Sys.time(), '%B %Y')`"
output: 
  vitae::awesomecv
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = FALSE, warning = FALSE, message = FALSE)
library(vitae)
library(tidyr)
library(dplyr)
library(readr)
library(glue)
library(tibble)
library(lubridate)
library(here)
library(scales)
```


# Qualifications and research interests

Highly effective computational system biologist, with graduate and post-doctoral work focusing on network inference and machine learning. Industry experience in development and implementation of computational tools for multi omics data analysis (including next-generation sequencing, metabolomics, proteomics), drug discovery, and target identification. I am interested in the application of machine learning/deep learning tools and techniques in the context of drug discovery, disease biology characterization, large data analytics for biology, while focused on bridging the gap between the computational and experimental labs through highly engaging and fruitful collaborations.


# Technical Skills
Machine learning, multi omics data analytics, R/Bioconductor, keras, perl, Latex, python, MATLAB, awk, bash. Adept user of OS X/macOS, Unix. Familiarity with cloud computing architectures (AWS) and high performance computing environments.


# Education
**<span style="font-size:1em;">Virginia Polytechnic Institute and State University</span>** \hfill _Blacksburg, VA_  
Ph.D. in Genetics, Bioinformatics, and Computational Biology \hfill _2007_

**<span style="font-size:1em;">Faculdade de Ciencias da Universidade de Lisboa</span>** \hfill _Lisboa, Portugal_  
B. Sc. in Biochemistry \hfill _2002_



# Experience
**<span style="font-size:1em;">Rheos Medicines</span>** \hfill _Boston, MA_  
Senior Director, Computational Biology and Bioinformatics \hfill _November 2020 - Present_  

I currently lead the Computational Biology and Bioinformatics group at Rheos Medicines. The group is focused on advancing Rheos' Precision Medicine platform, developing tools and capabilities to integrate multi-omics data that are fed into machine learning algorithms for patient stratification, target identification, and biomarker discovery. Some of the functions associated with the role: 

 - Implementation of Computational Biology and Bioinformatics strategy within the company
 - Managing expectations of the Executive team in terms of current capabilities as well as areas for growth in 1-5 year plans.
 - Management of team activities in line with cross-functional teams needs as well as development of internal capabilities to advance the Computational Biology and Bioinformatics function
 - Managing the development of the Machine Learning and Deep Learning capabilities at Rheos through internal advancement or through collaborations with external partners
 - Managing Computational Biology CRO engagement and deliverables

**<span style="font-size:1em;">Wyss Institute @ Harvard University</span>** \hfill _Boston, MA_  
Lead, Predictive BioAnalytics Initiative, Advanced Technology Team \hfill _July 2016 - November 2020_  

As the group lead of the Predictive BioAnalytics Initiative at the Wyss Institute, I led a team of computational biologists, computer scientists, and software engineers to address challenges in machine learning and biomedical data sciences, both in-house or thtough external collaborations. Some of the functions associated with the role included:

 - Development and implementation of research strategy for the Initiative, focusing on enabling ML/DL/AI capabilities
 - Managing and mentoring staff scientists, post-doctoral fellows, graduate students, and interns
 - Writing and managing federal grant applications and grant awards with DARPA, NIH
 - Hands-on development of algorithms and computational approaches for dissemination internally and with corporate partners via R Shiny applications
 - Responsible for the implementation of diverse tools for analysis of high throughput data (transcriptomics, RNA-seq, metabolomics, proteomics, 16S rDNA sequencing) in support of diverse grant work

**<span style="font-size:1em;">Evelo Biosciences</span>** \hfill _Cambridge, MA_  
Senior Scientist, Computational Systems Biology Lead \hfill _January 2015 - April 2016_  

As the lead Computational Systems Biologist at Evelo Biosciences, I was involved in the build out of the computational capabilities of the company, in support of preclinical development of microbiome-focused therapeutics for oncology. Some of the functions of the role included:

 - Implementation of diverse tools for analysis of high throughput data (transcriptomics, RNA-seq, metabolomics) as well as a 16S rDNA sequencing analysis pipeline
 - Responsible for the identification of novel therapeutic opportunities based on in-house and publicly available data
 - Implementation of analytical software tools to be used by bench scientists in R Shiny
 - Responsible for the interface with IT provider to delineate and expand computational capabilities of the company, from general to research needs
 
<!-- **<span style="font-size:1em;">Symbiota (now Indigo Ag)</span>** \hfill _Cambridge, MA_   -->
<!-- Consultant, Computational Systems Biology Lead \hfill _December 2014 - January 2015_   -->

<!-- In a short engagement with Symbiota, I developed an R Shiny application for pathway enrichment for plant transcriptomics data. -->


**<span style="font-size:1em;">Ember Therapeutics</span>** \hfill _Cambridge, MA_  
Principal Scientist, Computational Systems Biology Lead \hfill _January 2014 - December 2014_  

At Ember Therapeutics I was the Computational Systems Biology lead, where I oversaw the implementation and deployment of novel analytical tools for the identification of novel target opportunities for increased energy expenditure, not only from publicly available data but also from data coming from internal efforts. Some of the functions of the role included:

 - Responsible for the implementation of diverse tools for analysis of high throughput data (transcriptomics, RNA-seq, metabolomics)
 - Implementation of a knowledge based and data driven construction of screening libraries for recombinant proteins, small peptides, and RNAi therapeutic efforts
 - Implementation of analytical strategies (QC, statistical analyses, hit-calling) for high throughput screens
 - Responsible for the interface with IT provider to delineate and expand computational capabilities of the company, from general to research needs
 

**<span style="font-size:1em;">Pfizer, Inc</span>** \hfill _Cambridge, MA_  
Senior Research Scientist, Computational Sciences Center of Emphasis \hfill _January 2011 - January 2014_  

I was a member of the Computational Sciences Center of Emphasis at Pfizer, providing computational support across different preclinical programs at the organization, from cardiovascular disease to pain management and drug repositioning efforts. Some of the functions of the role included:

 - Development and implementation of a network analysis tool for the characterization of differential networks in healthy and diseased populations under the scope of metabolic diseases
 - Implementation of network inference tools for the analysis of large-scale data sets
 - Development and implementation of a methodology for metabolite set enrichment analysis for metabolomics data
 - Performed data analysis in transcriptomics, proteomics and metabolomics for different partners within the organization with particular emphasis in Cardiovascular and Metabolic diseases

 **<span style="font-size:1em;">Howard Hughes Medical Institute @ Boston University</span>** \hfill _Boston, MA_  
Post-doctoral Fellow \hfill _July 2007 - January 2011_ 
 
Post-doctoral training with Dr. James Collins at Boston University, focusing on the application of machine learning and network inference approaches in biomedicine. Some efforts included: 
 
 - Developed a network inference algorithm for identification of regulatory architectures of pathways
 - Identified a novel mechanism of action for antifungal drugs using transcriptomics and metabolomics data
 - Identified and characterized the small RNA regulatory network in bacterial systems using gene expression data
 - Performed data analysis of gene expression data and metabolomics data in bacterial and fungal systems

 
# Awards and Grants
**<span style="font-size:1em;">Molecular circuits in the hematopoietic stem cell niche (NIH)</span>** \hfill _$415,000_  
co-Investigator \hfill _9/1/20_  

**<span style="font-size:1em;">Synergistic Discovery and Design	(DARPA)</span>** \hfill _$2,000,000_  
co-PI \hfill _9/1/17_  


# Publications
 - Gazzaniga, FS, **Camacho, DM**, Wu, M, Palazzo, MFS, Dinis, ALM, Grafton, FN, Cartwright, MJ, Super, M, Kasper, DL, Ingber, DE (2021), Harnessing Colon Chip technology to identify commensal bacteria that promote host tolerance to infection, _Frontiers in Cellular and Infection Microbiology_, 11, 105 [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33777849/)]
 - Sperry, MM, Novak, R, Keshari, V, Dinis, ALM, Cartwright, MJ, **Camacho, DM**, Par√©, JF, Super, M, Levin, M, Ingber, DE (2021), Enhancers of host immune tolerance to bacterial infection discovered using linked computational and experimental approaches, _bioRxiv_ [[bioRxiv](https://www.biorxiv.org/content/10.1101/2021.09.30.462576v1)]
 - Bein, A, ,Fadel, CW, Swenor, B, Cao, W, Powers, RK, **Camacho, DM**, Naziripour, A, Parsons, AW, LoGrande, NT, Sharma, S, Kim, S, Jalili-Firoozinezhad, S, Grant, J, Breault, DT, Iqbal, J, Ali, A, Denson, LA, Moore, SR, Prantil-Baun, R, Goyal, G, Ingber, DE (2021), Nutritional deficiency recapitulates intestinal injury associated with environmental enteric dysfunction in patient-derived Organ Chips, _medRxiv_ [[medRix](https://www.medrxiv.org/content/10.1101/2021.10.11.21264722v1)]
 - Bojar, D, Powers, RK, **Camacho, DM**, Collins, JJ (2020), Deep-Learning Resources for Studying Glycan-Mediated Host-Microbe Interactions, _Cell Host & Microbe_, 29, 132-144 [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33120114/)]
 - Valeri, J, Collins, KM, Ramesh, P, Alcantar, M, Lepe, BA, Lu, TK, **Camacho, DM** (2020), Sequence-to-function deep learning frameworks for engineered riboregulators, _Nature Communications_, 11, 5058 [[PubMed](https://pubmed.ncbi.nlm.nih.gov/33028819/)]
 - Jalili-Firoozinezhad, S, Gazzaniga, FS, Calamari, EL, **Camacho, DM**, Fadel, C, Nestor, B, Cronce, MJ, Tovaglieri, A, Levy, O, Gregory, KE, Breault, DT, Cabral, JMS, Kasper, DL, Novak, R, Ingber, DE (2019),	A complex human gut microbiome cultured in an anaerobic intestine-on-a-chip, _Nature Biomedical Engineering_, 3, 520-531 [[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/31086325)]
 - Tovaglieri, A, Sontheimer-Phelps, A, Geirnaert, A, Prantil-Baun, R, **Camacho, DM**, Chou, DB, Jalili-Firoozinezhad, S, de Wouters, T, Kasendra, M, Super, M, Cartwright, M, Richmond, CA, Breault, DT, Lacroix, C, Ingber, DE (2019), Species-specific enhancement of enterohemorrhagic _E. coli_ pathogenesis mediated by microbiome metabolites, _Microbiome_, 7, 43 [[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/30890187)]
 - **Camacho, DM**, Collins, KM, Powers, RK, Costello, JC, Collins, JJ (2018), Next-generation machine learning for biological networks, _Cell_, 173, 1581-1592 [[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/29887378)]
 - Musah, S, Dimitrakakis, N, **Camacho, DM**, Church, GM, Ingber, DE (2018), Directed differentiation of human induced pluripotent stem cells into mature kidney podocytes and establishment of a Glomerulus Chip, _Nature Protocols_, 13, 1662-1685 [[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/29995874)]
 - Paandey, SP, Winkler, JA, Li, H, **Camacho, DM**, Collins, JJ, Walker, GC (2014), Central role for RNase YbeY in Hfq-dependent and Hfq-independent small-RNA regulation in bacteria, _BMC Genomics_, 15, 121 [[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/24511998)]
 - Galagan, JE, Minch, K, Peterson, M, Lyubetskya, A, Azzizi, E, Sweet, L, Gomes, A, Rustad, T, Dolganov, G, Glotova, I, Abeel, T, Mawhinney, C, Kennedy, A, Allard, R, Brabant, W, Krueger, A, Jaini, S, Honda, B, Yu, W-H, Hickey, M, Zucker, J, Garay, C, Weiner, B, Sisk, P, Stolte, C, Winkler, J, Van de Peer, Y, Iazzetti, P, **Camacho, D**, Dreyfuss, J, Liu, Y, Dorhoi, A, Mollenkopf, H-J, Drogaris, P, Lamontagne, J, Zhou, Y, Piquenot, J, Park, ST, Raman, S, Kaufmann, S, Mohney, R, Chelsky, D, Moody, B, Sherman, D, Schoolnik, G (2013), The Mycobacterium tuberculosis regulatory network and hypoxia, _Nature_, 499, 178-183 [[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/23823726)]
 - Belenky, P, **Camacho, D**, Collins, JJ (2013), Fungicidal drugs induce a common oxidative-damage cellular death pathway, _Cell Reports_, 3, 350-358 [[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/23416050)]
 - Marbach, D, Costello, JC, Kuffner, R, Vega, N, Prill, RJ, **Camacho, DM**, Allison, KR, the DREAM5 Consortium, Kellis, M, Collins, JJ, Stolovitzky, G (2012), Wisdom of crowds for robust gene network inference, _Nature Methods_, 9, 796-804 [[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/22796662)]
 - Dwyer, DJ, **Camacho, DM**, Callura, JM, Kohanski, MA, Collins, JJ (2011), Antibiotic-induced bacterial cell death exhibits physiological and biochemical hallmarks of apoptosis, _Molecular Cell_, 46, 561-572 [[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/22633370)]
 - Modi, SR, **Camacho, DM**, Kohanski, MA, Collins, JJ (2011), Functional characterization of bacterial sRNAs using a network biology approach, _Proc. Natl. Acad. Sci. USA_, 108, 15522-15527 [[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/21876160)]
 - **Camacho, DM**, Collins, JJ (2009), Systems biology strikes gold, _Cell_, 137, 24-26 [[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/19345182)]
 - **Camacho, D**, Vera-Licona, P, Laubenbacher, R, Mendes, P (2007), Comparison of existing reverse engineering methods by use of an in silico system, _Ann. N. Y. Acad. Sci._, 1115, 73-89 [[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/17925358)]
 - Mendes, P, **Camacho, D**, de la Fuente, A (2005),	Modelling and simulation for metabolomics data analysis, _Biochem. Soc. Trans._, 33, 1427-1429 [[PubMed](https://www.ncbi.nlm.nih.gov/pubmed/16246137)]
 - **Camacho, D**, de la Fuente, A, Mendes, P (2005), The origin of correlations in metabolomics data, _Metabolomics_, 1, 53-63 [[Link](https://link.springer.com/article/10.1007/s11306-005-1107-3)]
 - Martins, AM, **Camacho, D**, Shuman, J, Sha, W, Mendes, P, Shulaev, V (2004), A systems biology study of two distinct growth phases of _Saccharomyces cerevisiae_ cultures, _Curr. Genomics_, 5, 649-663 [[Link](https://www.ingentaconnect.com/content/ben/cg/2004/00000005/00000008/art00005)]


# Patents
 - Novak, R, Jalili-Firoozinezhad, S, Gazzaniga, FS, Calamari, EL, **Camacho, DM**, Nestor, BA, Fadel, C, Cronce, ML, Kasper, DL, Ingber, DE, Bein, A, Complex Human Gut Microbiome Cultured in an Anaerobic Human Gut-on-A-Chip (_submitted_)
 - Driggers, EM, Soh, K, **Camacho DM**, Transcriptional subseetting of patient cohorts based on metabolic pathway activity (_submitted_)
 - Garruss, AS,  Church, GM, **Camacho, DM**, Riboregulators and methods of use thereof (_submitted_)
 - Soenksen, LR, **Camacho, DM**, Collins, KM, Angenent-Mari, NM, Garruss, AS, Collins, JJ, Church, GM, Method for deep learning for RNA synthetic biology (_submitted_)
 - Collins, JJ, Belenky, P, **Camacho, DM** (2015), Compositions and methods for treating fungal infections, WO2014130922A1 [[Link](https://patents.google.com/patent/WO2014130922A1/en)]

# References
References will be provided upon request.
